test/k8sT/manifests/policy-stress-test.yaml

```diff
@@ -0,0 +1,1066 @@
+# Based on an automatically generated connectivity-check.
+# Edited manually.
```

Comment:
Small nit, it's not possible to generate this file? I'm just thinking this file might get outdated.

Reply:
For now, not possible, edits have been made locally. Will have to resolve later, if necessary.

Reply:
In general, the L7 policies in the connectivity-check were not elaborate enough to be useful for the stress test. Also had to add a label to satisfy the test manifest abstraction, but then used that same label in half of the generated identities and the CNPs as well. Will have to see if these changes are acceptable as part of examples/kubernetes/connectivity-check. Current test manifest helper requires yamls be placed within `test/k8sT/manifests/`, though.